### ISSUE TITLE
Design/Arrows for "merge people" dialogs should be unified

### ISSUE THREAD

**Issue by C-Otto** [immich-app/immich#11754]
### The bug

Currently, there are at least two different ways to merge two people into one.

When selecting the "Merge people" option, the resulting page shows:
![a](https://github.com/user-attachments/assets/10369c25-e22f-403a-9359-a016787d7723)

When changing the name to another existing person's name, a modal dialog appears:
![b](https://github.com/user-attachments/assets/129c3091-f212-4a3f-a980-05d73427a089)

The arrow in the upper variant is easy to understand, and the option to reverse direction can be inferred from the smaller symbol below.

In the lower variant the direction can be swapped by clicking on the icon, which is not intuitive. Furthermore, the icon itself does not convey order, it doesn't look like an arrow.

I suggest to use the upper style for both (all) dialogs/pages that are used to merge people.

See also #6382 and #8335.

### The OS that Immich Server is running on

Debian Stable

### Version of Immich Server

v1.111.0

### Version of Immich Mobile App

none

### Platform with the issue

- [ ] Server
- [X] Web
- [ ] Mobile

### Your docker-compose.yml content

```YAML
-
```

### Your .env content

```Shell
-
```

### Reproduction steps

```bash
-
```

**Comment by danieldietzler** [comment]
Hey, the merge direction has been made more intuitive in #12601. Do you consider this still an issue?

**Comment by C-Otto** [comment]
Yes, this issue is about visual communication via the arrows. Nothing changed about that.
